addons/web/static/src/scss/bootstrap_overridden.scss

```diff
@@ -282,10 +282,11 @@ $kbd-box-shadow: 0px 1px 1px rgba($o-black, 0.2), inset 0px -1px 1px 1px rgba($o
 $input-bg: $o-view-background-color !default;
 $input-focus-border-color: $o-brand-primary !default;
 $form-check-input-checked-color: $o-brand-lightsecondary !default;
-$form-check-input-checked-border-color: $o-brand-primary !default;
-$form-check-input-checked-bg-color: $o-brand-primary !default;
+$form-check-input-checked-border-color: $o-success !default;
```

Comment:
Sorry @chgo-odoo, I overlooked this.
Default checkboxes should remain `primary`, `$o-success` is supposed to be applied on switches only.

Could you reassign `$form-switch-[...]` variables instead?